### PR TITLE
Create morse decoder

### PR DESCRIPTION
**Pull request by Alejuda**
Hi there! In this branch, my coding partner @romans-adi and I collaborated to work on the following tasks:

- Set up Ruby linters for the project.
- Created a method to decode a Morse code character.
- Implemented a method to decode an entire word in Morse code.
- Added a method to decode the entire message in Morse code.

_**v.g the encrypted message was "A BOX FULL OF RUBIES"**_

### ♻️ Update for the second review
- Use the ternary operator in conditionals to improve readability. 
- change string concatenations to use the join method.
- replace print with puts to avoid the "%" error.
- Fix broken links on the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,184 @@
...
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make your changes and commit them with descriptive commit messages.
+4. Push your changes to your forked repository.
+5. Submit a pull request to the main repository, explaining your changes in detail.
+
+Please adhere to the coding conventions and guidelines specified in the project.
+
+Feel free to check the [issues page](../../issues/).
```
Comment (abdulhamiid) [inline README.md]: image

Please fix the broken link to the issue page in your Readme for a professional readme 🤗

File: README.md [hunk 2]
```diff
@@ -1 +1,184 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## Support <a name="support"></a>
+
+If you encounter any issues or have any questions or suggestions, please open an issue on the [issue tracker](../../issues/).
```
Comment (abdulhamiid) [inline README.md]: image

Please fix the broken link to the issue page

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️Run a program using the command line.
- ✔️ Descriptive readme
- ✔️ Use Ruby syntax for basic programming operations.
- ✔️ Use Ruby to solve basic coding challenges.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @Alejuda and @romans-adi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**